app/views/estimatedIncomeTax/DetailedIncomeTaxEstimateView.scala.html

```diff
@@ -33,148 +33,173 @@
+    displayBackLink = false,
+    mainHeadingText = Messages("tai.estimatedIncome.detailedEstimate.heading"),
+    preHeadingText = taxYearRangeString,
+    preHeadingAccessibleAnnouncement = Messages("tai.estimatedIncome.accessiblePreHeading"),
+    showGdsVersion = true
     )
 }
 
 @mainTemplate(
     title = Messages("tai.estimatedIncome.detailedEstimate.title"),
-    articleClasses = Some("full-width"),
-    pageHeader = Some(header),
-    authedUser = Some(user)
+    pageTitle = Some(Messages("tai.service.navTitle")),
+    authedUser = Some(user),
+    backLinkContent = Some(Messages("tai.back-link.upper")),
+    backLinkId = "backLink"
 ){
-    <div class="grid-row">
-        <div class="grid grid-2-3">
-            <div class="inner-block">
-                <div class="soft--top flush--bottom">
-                    <div class="grid-row">
-                        <div class="column-one-half">
-                            <h2 class="flush--bottom">
-                                @Messages("tai.incomeTax.totalIncomeTaxEstimate")
-                                <span id="total-income-tax-estimate" class="display-block heading-48 section faded-text flush section--narrow">
-                                        @pounds(viewModel.incomeTaxEstimate)
-                                </span>
-                            </h2>
-                        </div>
-                    </div>
-                    @viewModel.selfAssessmentAndPayeText.map(paragraphText =>
-                        <p>{paragraphText}</p>
-                    )
-                </div>
-
-                <div class="section soft--top">
-                    <h2 class="heading-section">
-                        @viewModel.taxOnIncomeTypeHeading
-                    </h2>
-                    <p id="taxOnEmploymentIncomeDesc">
-                        @Html(viewModel.taxOnIncomeTypeDescription)
-                    </p>
-
-                    @taxBandTable(
-                        tableClass = "employmentIncomeTaxDetails",
-                        taxBands = viewModel.nonSavings,
-                        taxRegion = viewModel.taxRegion)
-                </div>
-
-                @if(viewModel.dividends.nonEmpty){
-                    <div class="section soft--top">
-                        <h2 id="tax-on-dividend-income-text" class="heading-section">@Messages("tai.estimatedIncome.detailedEstimate.dividendIncome.subHeading")</h2>
-                        <p id="tax-on-dividend-income-desc">
-                            @Messages("tai.estimatedIncome.dividend.para.desc",
-                            MoneyPounds(viewModel.totalDividendIncome, 0).quantity,
-                            MoneyPounds(viewModel.taxFreeDividendAllowance, 0).quantity)
-                        </p>
-                        @taxBandTable(tableClass = "employmentIncomeTaxDetails", taxBands = viewModel.dividends, taxRegion = viewModel.taxRegion)
-                    </div>
-                }
+<style>
```

Comment:
why use Internal Styles ?
we can use available GDS classes to control margin and padding eg:   govuk-!-margin-  &  govuk-!-padding-1

and/ or sometimes put CSS in tai-css file

Reply:
the first one didn't work and the second I wanted your opinion. thank you!